scripts/check-formatting.sh

```diff
@@ -5,15 +5,19 @@
 source scripts/find_clang_format.sh
 find_clang_format
 
-files='.*\.\(cpp\|hpp\|cc\|cxx\|h\)'
+echo "Checking format with ${format}"
 
-output="$(diff -u <(find asgard/ -regex "$files" -exec cat {} \;) <(find asgard/ -regex "$files" -exec ${format} -style=file {} \;))"
+files_type=".*\.\(cpp\|hpp\|cc\|cxx\|h\)"
+files=$(find asgard/ -regex ${files_type})
+output="$(clang-format --dry-run ${files} 2>&1)"
```

Comment:
```suggestion
output="$(${format} --dry-run ${files} 2>&1)"
```
clang-format is found automatically by the script `find_clang_format` and stored in ${format}

Reply:
:+1: 
True story